src/bootstrap/bootstrap.py

```diff
@@ -460,8 +460,14 @@ def download_stage0(self):
 
     def downloading_llvm(self):
         opt = self.get_toml('download-ci-llvm', 'llvm')
+        supported_platforms = [
+            "x86_64-unknown-linux-gnu",
+            # FIXME: maybe windows-gnu should be supported too?
+            "x86_64-pc-windows-msvc",
+            "x86_64-apple-darwin",
+        ]
         return opt == "true" \
-            or (opt == "if-available" and self.build == "x86_64-unknown-linux-gnu")
+            or (opt == "if-available" and self.build in supported_platforms)
```

Comment:
`config.toml` should be adjusted to make it apparent `if-available` is an option. Right now the default is `false` and the possible options are not listed, so it is kind of implied that `true` is the only valid option there.

Reply:
furthermore `config.toml` should be adjusted to remove note that only `x86_64-unknown-linux-gnu` is supported by that option.

Reply:
It's there in `defaults/config.compiler.toml`, which I think most people are using by now, but I agree it should be in config.toml.example too. Updated.